spec/apis/single-spa-events-api.spec.js

```diff
@@ -73,51 +73,47 @@ describe(`events api :`, () => {
       boom = false;
     });
 
-    it(`is fired on the window whenever the hash changes`, (done) => {
-      singleSpa
-        .triggerAppChange() // start with a clean slate (no previous tests doing anything)
-        .then(() => {
-          window.addEventListener("single-spa:routing-event", finishTest);
-          window.location.hash = `#/hash-was-changed`;
-
-          function finishTest() {
-            window.removeEventListener("single-spa:routing-event", finishTest);
-            done();
-          }
-        })
-        .catch(fail);
+    it(`is fired on the window whenever the hash changes`, async () => {
```

Comment:
I'm pretty sure you can still do `async (done) =>` if that makes things easier. That way you don't have to do the `finish` promise trick

Reply:
No, you can't, according to https://jestjs.io/blog/2021/05/25/jest-27#features-coming-with-breaking-changes